apps/browser/qml/pages/components/TabGridView.qml

```diff
@@ -0,0 +1,153 @@
+/****************************************************************************
+**
+** Copyright (c) 2014 - 2019 Jolla Ltd.
+** Copyright (c) 2019 - 2021 Open Mobile Platform LLC.
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.1
+import Sailfish.Silica 1.0
+import org.nemomobile.configuration 1.0
+import "." as Browser
+
+SilicaGridView {
+    id: tabView
+
+    property bool portrait
+    property bool privateMode
+    property bool closingAllTabs
+    property bool loaded
+
+    property var remorsePopup
+    readonly property bool largeScreen: Screen.sizeCategory > Screen.Medium
+    readonly property real thumbnailHeight: largeScreen
+                                            ? Screen.width / 3
+                                            : !portrait ? parent.height / 2.5 : parent.width / 1.66
+    readonly property real columns: largeScreen
+                                        ? portrait ? 2 : 3
+                                        : parent.width < 2 * parent.height
+                                          ? parent.width <= height ? 1 : 2 : 3
+    readonly property real thumbnailWidth: (parent.width - Theme.horizontalPageMargin * 2 - (Theme.paddingLarge * (columns - 1))) / columns
+
+    signal hide
+    signal enterNewTabUrl
+    signal activateTab(int index)
+    signal closeTab(int index)
+    signal closeAll
+    signal closeAllCanceled
+    signal closeAllPending
+
+    onCountChanged: if (count > 0) closingAllTabs = false
+    onClosingAllTabsChanged: if (closingAllTabs) closeAllPending()
+
+    width: parent.width - Theme.horizontalPageMargin
+    height: parent.height
+    y: Theme.paddingLarge
```

Comment:
Shifting the `SilicaGridView` down by `Theme.paddingLarge` here is unfortunately causing the pulley menu to be shifted down as well, so there's a gap at the top when you pull the items down.

I recommend removing this `y` shift here, and instead adding `Theme.paddingLarge` to both `TabGridView.header` items in `TabView.qml`.
```suggestion
```

Reply:
done